portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminTaxonomyAPI/ExposeErcFromAssetLibrariesForVocabularies.testcase

```diff
@@ -159,6 +159,42 @@ definition {
 		}
 	}
 
+	@priority = "4"
+	test CanRecieveCorrectErcValueInCategory {
```

Comment:
The test looks good. But I'm wondering should we create a new test file for category since we haven't covered functional tests for LPS-131093(Add support for ERC in TaxonomyCategory)? @magjed4289 @Hijky

Reply:
good observation @yanan! Yeah, looks like the endpoints that are being checked here belong to the TaxonomyCategory which is a separate section from TaxonomyVocabulary. Would be nice to have a separate testcase file, but I think a must here is to move the macros to a separate file. 
<img width="1212" alt="image" src="https://user-images.githubusercontent.com/35199951/186841898-9b9a9b6e-5f39-4a03-b241-de9093024d2d.png">